edk2toolext/tests/test_pyopenssl_signer.py

```diff
@@ -7,7 +7,223 @@
 ##
 import unittest
 from edk2toolext.capsule import pyopenssl_signer
-from OpenSSL import crypto
+
+# TEST_CERT is a cert used to test the ability to parse pfx and pull out necessary information.
+# To generate a new test cert follow these steps:
+# 1. Generate the Cert (.pfx) file'
+#   There are multiple ways to do this, but running this powershell script is the easiest:
```

Comment:
can this be made a docstring using ''' so that it is easier to copy and paste

Reply:
Updated to docstring